templates/tftmpl/tmplfunc/services_regex.go

```diff
@@ -201,7 +201,7 @@ func (d *servicesRegexQuery) Fetch(clients dep.Clients) (interface{}, *dep.Respo
 	// Without this delay, CatalogServices may have services that are not yet
 	// propagated to HealthServices as healthy services since services are initially
 	// set as critical. https://www.consul.io/docs/discovery/checks#initial-health-check-status
-	time.Sleep(1 * time.Second)
+	time.Sleep(10 * time.Second)
```

Comment:
Any reason behind increasing the sleep duration by 10x? Wondering if there is a way to deterministically figure out rather than following a trial and error based approach

Reply:
Tried on 5x. Issue was still seen.
Could not thing of any other way. 😢